scrapers/nd/bills.py

```diff
@@ -1,164 +1,125 @@
 import logging
 import re
-import requests
-import lxml.html
-import datetime as dt
-from .actions import NDCategorizer
+from dateutil import parser
 from openstates.scrape import Scraper, Bill
-from spatula import HtmlListPage, HtmlPage, XPath, CSS
-
-
-def get_committee_names(session):
-    source = f"https://www.ndlegis.gov/assembly/{session}/committees"
-    response = requests.get(source)
-    content = lxml.html.fromstring(response.content)
-    committee_names = set()
-    committees = content.xpath(".//div[@class='grouping-wrapper']//a/text()")
-    for committee in committees:
-        committee_names.add(committee.strip())
-    return committee_names
+from spatula import JsonPage
+from .actions import NDCategorizer
 
 
-class BillList(HtmlListPage):
+class BillList(JsonPage):
+    # TODO: remove this hard-coded {assembly-session_year} below, there must be
+    #  a way to get access to the "start_date" value for the active leg session
+    #  being scraped, so that the year could be pulled from that and this could
+    #  then be assembled in a variable way in the `NDBillScraper()` class.
```

Comment:
@NewAgeAirbender curious for your thoughts on how this could be achieved.

Reply:
I know that the active sessions' `identifier` value (from the dictionary in the `legislative_sessions` list in the `__init__` file) is accessible within the `NDBillScraper`, as it's getting passed to its `scrape()` method as the `session` argument.

I began looking more into OS-core to try to find an answer to how those other values in the active session's dictionary can be accessed. Didn't find answer as of last night, but I gotta imagine there are other scrapers that pass more than just the session identifier, right?

Reply:
@chrisyamas it's kinda hacky, but to get that value you can do --

```python
        for i in self.jurisdiction.legislative_sessions:
            if i["identifier"] == session:
                start_date = i["start_date"]
```

Reply:
Here's a few cases in the existing code where we do variations on that --

https://github.com/search?q=repo%3Aopenstates%2Fopenstates-scrapers%20self.jurisdiction&type=code

Reply:
Amazing, that's what I didn't realize, that I had access to `self.jurisdiction`. Thanks, @showerst! I'll incorporate that in my next commit.